beacon-chain/types/crystallized_state.go

```diff
@@ -77,16 +30,16 @@ func NewGenesisCrystallizedState(genesisJSONPath string) (*CrystallizedState, er
 	var err error
 	if genesisJSONPath != "" {
 		log.Infof("Initializing crystallized state from %s", genesisJSONPath)
-		genesisValidators, err = initialValidatorsFromJSON(genesisJSONPath)
+		genesisValidators, err = casper.InitialValidatorsFromJSON(genesisJSONPath)
```

Comment:
I think that file I/O should be outside of both `casper` and `types` packages. We can load the file in `node.go`, and keep the logic out of here.